# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/browser/browser.toml
# Commit: af0f14eed70e
# Full Hash: af0f14eed70edb2abb1ecd3cae4928387287d3e1
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2024-06-14 20:49:45
# Regressor Bug: 1901990
# File Overlap Count: 2
# Description:
#   Bug 1901990 - Introduce a pref for setting a mock HTTPS RR, r=necko-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D213559
# ==============================================================================

diff -r 9f913cbd60ee -r af0f14eed70e netwerk/test/browser/browser.toml
--- a/netwerk/test/browser/browser.toml	Thu Jun 13 15:08:04 2024 +0200
+++ b/netwerk/test/browser/browser.toml	Fri Jun 14 09:11:49 2024 +0000
@@ -205,3 +205,9 @@
 ]
 
 ["browser_test_offline_tab.js"]
+
+["browser_mock_https_rr.js"]
+skip-if = [
+  "http3",
+  "http2",
+]